### ISSUE TITLE
Nvda does not read the options inside the combo box to choose the ways to confirm the microsoft account

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7100]
I do not know how to point you to reproof.
I'm using nvda master 13996, and windows 10 15063 64bits.
In the settings center there was a request for me to check the microsoft account on this device so that I could use some microsoft cincronization options.
When I opened the notification, the windows settings app was opened, where a combo box was displayed so I can choose how I would like to receive the microsoft account verification code, by mobile phone, text message, or by email.
The error occurred when I pressed alt + down arrow and walked through the options with the arrows up and down, nvda just made an error sound and did not read the options.
I restarted the nvda without complements but the error percistiu.
The following is a log of the time of error:

Input: kb(laptop):downArrow
ERROR - eventHandler.executeEvent (21:50:00):
error executing event: gainFocus on <NVDAObjects.Dynamic_EdgeNodeListItemUIA object at 0x0A2AC9D0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in __init__
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\wintenApps\globalPlugins\wintenObjs\__init__.py", line 199, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\focusHighlight\globalPlugins\focusHighlight.py", line 621, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\__init__.py", line 410, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\enhancedTouchGestures\globalPlugins\enhancedTouchGestures.py", line 94, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\easyTableNavigator\globalPlugins\easyTableNavigator.py", line 82, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\AudioThemes\globalPlugins\AudioThemes\__init__.py", line 72, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\ferna\AppData\Roaming\nvda\addons\Favorite folders\globalPlugins\FavoriteFolders\__init__.py", line 161, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "browseMode.pyc", line 1342, in event_gainFocus
  File "textInfos\__init__.pyc", line 311, in isOverlapping
  File "treeInterceptorHandler.pyc", line 180, in compareEndPoints
  File "NVDAObjects\UIA\__init__.pyc", line 625, in compareEndPoints
COMError: (-2146233079, None, (None, None, None, 0, None))
IO - inputCore.InputManager.executeGesture (21:50:02):
Input: kb(laptop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (21:50:02):
Developer info for navigator object:
name: u'Email fe*****@abril.com.br\u200e'
role: ROLE_LISTITEM
states: STATE_FOCUSABLE, STATE_SELECTABLE, STATE_FOCUSED, STATE_SELECTED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_EdgeNodeListItemUIA object at 0x0A2AC9D0>
Python class mro: (<class 'NVDAObjects.Dynamic_EdgeNodeListItemUIA'>, <class 'NVDAObjects.UIA.edge.EdgeNode'>, <class 'NVDAObjects.UIA.ListItem'>, <class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: ''
location: (152, 341, 380, 21)
value: None
appModule: <'wwahost' (appName u'wwahost', process ID 1448) at address 4ef4690>
appModule.productName: u'Sistema Operacional Microsoft\xae Windows\xae'
appModule.productVersion: u'10.0.15063.0'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 2097874
windowClassName: u'Windows.UI.Core.CoreComponentInputSource'
windowControlID: 0
windowStyle: 1342177280
windowThreadID: 11692
windowText: u'CoreInput'
displayText: u''
UIAElement: <POINTER(IUIAutomationElement) ptr=0x72dabc8 at a2fea80>
UIA automationID: 
UIA frameworkID: MicrosoftEdge
UIA runtimeID: (42, 2097874, 3, 1054)
UIA providerDescription: [pid:1448,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:edgehtml.dll)]
UIA className:

**Comment by Adriani90** [comment]
@fernando-jose-silva do you think this issue is still present? The interface has been changing from time to time in Windows 10. I cannot really find the combo box you describe. Could you please post exact steps to reproduce? Thanks.

**Comment by fernando-jose-silva** [comment]
I also would not know how to reproduce the problem, I came to this windows configuration screen from a windows notification.
As I have never encountered this problem I believe that the call can be closed.
Thank you very much.

**Comment by Adriani90** [comment]
Closing.
